docs/source/examples/petstore/petstore.all.rego

```diff
@@ -4,6 +4,13 @@ default allow = false
 
 default deny = false
 
+allow {
+	seal_list_contains(input.subject.groups, `banned`)
```

Comment:
why do we need to introduce seal_list_contains() function, why not just:
input.subject.groups[_] == "banned"

Reply:
also shouldn't it be:
    seal_subject.groups[_] == "banned"
based on the decoded jwt below?

Reply:
seal_list_contains is not introduced here in this pr.  seal_list_contains is a generic rego function defined by seal to perform a generic look for a member of a list.

seal_list_contains(input.subject.groups, `banned`) is not dealing with the where clause use of the rule.  it's dealing with the `subject` clause:
`subject group banned`

Reply:
i'm going to raise another issue to change `input.subject` to use the seal_subject.  let's not conflate that fix here.

Reply:
https://github.com/infobloxopen/seal/issues/82

Reply:
Is it ok to implement #82 in different PR?